delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/ResourcesImplSpec.scala

```diff
@@ -346,6 +348,14 @@ class ResourcesImplSpec
           mkResource(expectedData).copy(rev = 3)
       }
 
+      "succeed with a schema update" in {
+        val updated      = source.removeKeys(keywords.id) deepMerge json"""{"number": 70}"""
+        val expectedData = ResourceGen.resource(myId2, projectRef, updated, Revision(schema3.id, 1))
+        resources
+          .update(myId2, projectRef, Some(schema3.id), 3, updated)
+          .accepted shouldEqual mkResource(expectedData).copy(rev = 4)
+      }
```

Comment:
Another test where the actual desired outcome is the schema constraining the resource, not sure if it's testable from here

Reply:
Should be better now?